deploy/docs/Installation_with_Helm.md

```diff
@@ -306,3 +310,21 @@ kubectl delete secret sumologic
 ```
 
 and the associated hosted collector can be deleted in the Sumo Logic UI.
+
+### Removing the kubelet Service
+
+The Helm chart uses the Prometheus Operator to manage Prometheus instances.
+This operator creates a Service for scraping metrics exposed by the kubelet, which isn't removed by the chart uninstall process.
+This Service is largely harmless, but can cause issues if a different release of the chart is installed, resulting in
+duplicated metrics from the kubelet.
+See [this issue](https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/1101) for a more detailed explanation.
```

Comment:
Let's create the issue upstream regardless and add a link here.

Reply:
Done.